### DISCUSSION TITLE
Powershell Right Format Prompt

### DISCUSSION THREAD

**Discussion by its-19818942118** [starship/starship#5540] [Q&A] [answered]
Hi, so I have been using starship on powershell for a while now 
but I have hit a bottle neck with customization with right format as  it isnt supported
i want time and cmd duration to be moved to the right someway with workaround. If anyone knows pls help
![image](https://github.com/starship/starship/assets/87006850/2b5816bc-836c-4ec9-a497-45625f93acab)

**Comment by pwnwriter** [comment]
Did you mean, something like this? 

![screenshot_2023-11-01_17-46-40](https://github.com/starship/starship/assets/90331517/452ef236-cb9d-44f4-8c34-47def90080cc)

just move your $variables to right like this

![screenshot_2023-11-01_17-47-48](https://github.com/starship/starship/assets/90331517/17f978c7-1f0a-41ca-b1c1-7422f0b59ae6)

**Comment by davidkna** [comment, marked as answer]
Support for `right_format` in PowerShell is tracked in issue #3177. To show modules on the right side of the upper line you can use the `$fill` module.

**Comment by its-19818942118** [comment]
thanks i will try it out
thanks for the fast discussion help

**Reply by OlaoluwaM** [reply]
How did you get this character to render btw?

![image](https://github.com/starship/starship/assets/37044906/ec89508a-b807-4d11-96b0-598fce0163fa)

**Reply by its-19818942118** [reply]
> How did you get this character to render btw?
> 
> ![image](https://github.com/starship/starship/assets/37044906/ec89508a-b807-4d11-96b0-598fce0163fa)
> 

Hey so sorry for the late reply
It basically moving the prompt to a new line
The first line having the first edge
Second line having the second edge hope this helps
